brainio_collection/packaging.py

```diff
@@ -9,7 +9,7 @@
 from tqdm import tqdm
 import numpy as np
 from PIL import Image
-import cv2
+#import cv2
```

Comment:
undo this deletion

Reply:
Yes, you're right (sorry, I forgot to delete it in the end)! Can you or Tiago do it here?

Reply:
thanks!
@tiagogmarques can you delete the line, and also delete it from the [`setup.py`](https://github.com/brain-score/brainio_collection/blob/992ae550d38681843cabfc37509d540acc44c8f6/setup.py#L22)?

Reply:
I don't find it in the setup.py. Is it part of the opencv-python package?

Reply:
Yep, it's opencv-python.